src/libs/actions/IOU.ts

```diff
@@ -480,6 +480,28 @@ type CreateTrackExpenseParams = {
     transactionParams: TrackExpenseTransactionParams;
 };
 
+type BuildOnyxDataForInvoiceParams = {
+    chatReport: OnyxEntry<OnyxTypes.Report>;
+    iouReport: OnyxTypes.Report;
+    transaction: OnyxTypes.Transaction;
+    chatCreatedAction: OptimisticCreatedReportAction;
+    iouCreatedAction: OptimisticCreatedReportAction;
+    iouAction: OptimisticIOUReportAction;
+    optimisticPersonalDetailListAction: OnyxTypes.PersonalDetailsList;
+    reportPreviewAction: ReportAction;
+    optimisticPolicyRecentlyUsedCategories: string[];
+    optimisticPolicyRecentlyUsedTags: OnyxTypes.RecentlyUsedTags;
+    isNewChatReport: boolean;
+    transactionThreadReport: OptimisticChatReport;
+    transactionThreadCreatedReportAction: OptimisticCreatedReportAction | null;
+    policy?: OnyxEntry<OnyxTypes.Policy>;
+    policyTagList?: OnyxEntry<OnyxTypes.PolicyTagLists>;
+    policyCategories?: OnyxEntry<OnyxTypes.PolicyCategories>;
+    optimisticRecentlyUsedCurrencies?: string[];
+    companyName?: string;
+    companyWebsite?: string;
+};
```

Comment:
Can we please group together all chat, iou, policy, etc, related params like we do for `MoneyRequestOptimisticParams`?